### ISSUE TITLE
ChatGPT - detect questions in chit_chat

### ISSUE THREAD

**Issue by Zabuzard** [Together-Java/TJ-Bot#815]
We want ChatGPT to check all messages in `#chit_chat` and let it figure out whether its a programming related question.

If so, it should hint the user at asking in `#questions` instead (has to be friendly though!).

This feature is kinda controversial and we need a quick way to disable it via config. So this needs some config toggle.

The feature needs fine tuning:
* dont trigger for very small messages (likely not a question to begin with)
* neither for very long messages (expensive)
* have a cooldown per user, so that people sending lots of messages wont constantly trigger it, but only once per 30 seconds maybe (Caffeeine cache would work)

**Comment by ankitsmt211** [comment]
i think we might wanna put this one on hold atm or drop, since `transfer-feature` handles this part.
